src/commons/utils.jsx

```diff
@@ -27,3 +28,203 @@ export function participantToString(participant, t, accounts) {
 	}
 	return participant?.fullName || participant?.name || participant?.address || '';
 }
+
+export const getFileExtensionAndColor = (file, theme) => {
```

Comment:
instead of switching like this, why not generate a number from the string to get a deterministic number without keeping a list?
```

function calcColor(label: string): keyof ThemeObj['avatarColors'] {
	let sum = 0;
	// eslint-disable-next-line no-plusplus
	for (let i = 0; i < label.length; i++) {
		sum += label.charCodeAt(i);
	}
	return `avatar_${(sum % 50) + 1}`;
}
```
as an example, this is the function that the same in the ds for the avatars

Reply:
@zovomat it was decided to show color depending on MIME TYPE

Reply:
that's what i mean, using the mime type as seed gives you consistent deterministic strings and it doesn't depend on the filename

Reply:
ok got your point.I'll update it